app.js

```diff
@@ -1 +1,43 @@
-console.log("js file is connected")
\ No newline at end of file
+const rootUrl = "https://api.edamam.com/search";
+const recipeUrlElement = document.querySelector("#recipe-url");
+const labelElement = document.querySelector("#label");
+const imageElement = document.querySelector("#cake-img");
+let ingredientsListElement = document.querySelector(".ingredients");
+
+const apiUrl = `${rootUrl}?q=birthday+cake+strawberry&app_id=589ecbd6&app_key=6d6116bfcbdc60fe641222727dc9eb8f`;
```

Comment:
You don't need this variable. Delete it.

Reply:
ingredientsListElement, I do need it, is the parent of the list of the recipe, and it changes in searchIngredient()--- not delete it!

Reply:
it is a const

Reply:
It was about `apiUrl`, when you change the `fetchRecipies` call to `searchIngredient`, then you don't need a separate link for initializing your UI with the details of Strawberry Cake, `searchIngredient` call will do it for you.
![Screenshot 2020-10-27 at 14 40 47](https://user-images.githubusercontent.com/7361924/97309586-81e40c80-1862-11eb-9b0c-aa64f61df1d4.png)